app/models/card.rb

```diff
@@ -34,14 +35,19 @@ def update_from_pack(user, card_params)
   end
 
   def verify_translation(user_translation)
-    if transform_string(original_text) == transform_string(user_translation)
-      update(review_date: DateTime.now + TIME_INTERVALS[correct_counter])
-      self.increment!(:correct_counter)
+    result = Text::Levenshtein.distance(transform_string(original_text), transform_string(user_translation))
+    if result < 3
+      if self.correct_counter == MAX_TIME_INTERVAL
+        update(review_date: DateTime.now + TIME_INTERVALS[MAX_TIME_INTERVAL - 1])
+      else
+        update(review_date: DateTime.now + TIME_INTERVALS[correct_counter])
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.